### PR TITLE
fixed fund fuse

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/components/wizard/index.jsx [hunk 1]
```diff
@@ -1,27 +1,14 @@
-import React, { useState, useEffect, useMemo } from 'react'
+import React, { Fragment, useEffect, useMemo } from 'react'
```
Comment (leonprou) [inline dapp/src/components/wizard/index.jsx:1]: I think you don[t need `Fragment` (adviced to use `<>`)

File: dapp/src/components/wizard/pages/ChooseCurrencyType.jsx [hunk 2]
```diff
@@ -43,8 +49,35 @@ const CurrencyOption = ({ logo, name, value }) => {
...
+  }
+
+  useEffect(() => {
+    if (jobId) {
+      setDelay(CONFIG.web3.bridge.pollingTimeout)
+    }
+  }, [jobId])
+
+  useEffect(() => {
+    if (fundingStatus) {
```
Comment (leonprou) [inline dapp/src/components/wizard/pages/ChooseCurrencyType.jsx:73]: I think you also should check here that the job is done (status is`success ` or `failed`)

**Review by leonprou — CHANGES_REQUESTED** [review]
Looks good, please see the comments

File: dapp/src/components/wizard/pages/ChooseCurrencyType.jsx (hunk 2 above)
Comment (LiorAgnin) [reply dapp/src/components/wizard/pages/ChooseCurrencyType.jsx:73]: If it isn’t falsy that means I received the response. In the render, I check his value

Comment (leonprou) [reply dapp/src/components/wizard/pages/ChooseCurrencyType.jsx:73]: So you receive a funding object (though it's the job)? Can it be received with a pending status, and then you stop fetchig before the funding is finished?

Comment (LiorAgnin) [reply dapp/src/components/wizard/pages/ChooseCurrencyType.jsx:73]: Take a look into `[fetchFundingStatus](https://github.com/fuseio/fuse-studio/blob/master/dapp/src/sagas/user.js#L123)` it updates the `fundingStatus` only if it `success` or `failed`

Comment (leonprou) [reply dapp/src/components/wizard/pages/ChooseCurrencyType.jsx:73]: Got it, cool

**Pull request merged**